bom-internal/pom.xml

```diff
@@ -678,6 +678,36 @@ POM as their parent.
                 <artifactId>localstack-utils</artifactId>
                 <version>0.2.22</version>
                 <scope>test</scope>
+                <exclusions>
+                    <exclusion>
+                        <groupId>com.amazonaws</groupId>
+                        <artifactId>aws-java-sdk-kinesis</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>com.amazonaws</groupId>
+                        <artifactId>aws-java-sdk-simpleworkflow</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>com.amazonaws</groupId>
+                        <artifactId>aws-java-sdk-dynamodb</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>com.amazonaws</groupId>
+                        <artifactId>aws-java-sdk-s3</artifactId>
```

Comment:
This one is potentially problematic. I would think that `toolbackup` in dockstore-support may potentially use this. 
Then again, this looks like the v1 api rather than the v2. 
What did you encounter that prompted this change?